codespell_lib/_codespell.py

```diff
@@ -663,6 +676,17 @@ def main(*args):
         parser.print_help()
         return EX_USAGE
 
+    if options.ignore_regex:
+        try:
+            ignore_word_regex = re.compile(options.ignore_regex)
+        except re.error as err:
+            print("ERROR: invalid regular expression \"%s\" (%s)" %
```

Comment:
Perhaps add which regex it is given we've already got the word regex and may have more if you add the URI stuff in.

Reply:
Done (will comment back on the issue to make sure thinking about URI flags matches)